content/docs/preferences.md

```diff
@@ -119,20 +119,27 @@ Select the name of the filetype in the left column; select one of the extensions
 
 To unregister an extension, click it in the right column, and use the left arrow `←` to remove it from the Registered Extensions column.
 
+For this section to work, please run Notepad++ in Administrator mode, since it is modifying the registry.
+
```

Comment:
I'd recommend putting this as the first sentence of this section, or at least adding a bold "**Note**" at the beginning of this line. Probably shouldn't assume users will read the entire section clear to the last line. So I'd suggest calling this part out in particular since this caveat of "this will only work under specific conditions" is important.